modules/client/src/addresslistVoting/internal/constants.ts

```diff
@@ -14,3 +15,54 @@ export const AVAILABLE_FUNCTION_SIGNATURES: string[] = [
     "removeAddresses",
   ).format("minimal"),
 ];
+
+export const INSTALLATION_ABI: MetadataAbiInput[] = [
```

Comment:
Is there a way to make this be imported from the plugin metadata?
I guess that the npm package doesn't export that?

Reply:
That would be the best solution, atm the `osx-ethers` does not provide this constant, but last publish was 2 months ago

Reply:
OS-486 created